Gems/RobotecSplineTools/Code/Source/Clients/SplinePublisher.cpp

```diff
@@ -0,0 +1,149 @@
+#include "SplinePublisher.h"
+
+#include <ROS2/Frame/ROS2FrameComponent.h>
+#include <ROS2/ROS2Bus.h>
+#include <ROS2/Utilities/ROS2Names.h>
+
+namespace SplineTools
+{
+    SplinePublisherConfiguration::SplinePublisherConfiguration()
+    {
+        m_TopicConfig.m_type = "nav_msgs::msg::Path";
+        m_TopicConfig.m_topic = "spline_path";
```

Comment:
Nit: Technically, the "spline path" might be confusing, because the spline doesn't move along the route defined by a path (like "tractor path" for example). We could use just `spline` or something like `spline_points`.

```suggestion
        m_TopicConfig.m_topic = "spline";
```